include/deal.II/base/vectorization.h

```diff
@@ -627,10 +633,14 @@ vectorized_transpose_and_store(const bool                            add_into,
 }
 
 
+//@}
+
+#ifndef DOXYGEN
```

Comment:
You exclude all of the specializations of the class from the doxygen generation. Is this on purpose?

Reply:
Yes - we never generated any official documentation where these variants get picked up, see for example [1]. There is also no additional information in the documentation of the potentially 8 specializations.

[1] https://www.dealii.org/developer/doxygen/deal.II/classVectorizedArray.html

Reply:
OK. Good to merge then!